base-notebook/Dockerfile

```diff
@@ -81,72 +83,65 @@ RUN echo "auth requisite pam_deny.so" >> /etc/pam.d/su && \
     fix-permissions "${CONDA_DIR}"
 
 USER ${NB_UID}
-ARG PYTHON_VERSION=default
+
+# Pin versions here, or set them to "default"
+ARG PYTHON_VERSION=3.9
+ARG MAMBA_VERSION=default
+ARG MICROMAMBA_VERSION=default
```

Comment:
Is this flexibility needed?

Reply:
Awesome. The builds no longer hang.

Sure, we can hardcode the default versions. Would you like me to do it, or are you on it?

Reply:
I'll do it, no worries :)

Reply:
Done